examples/step-45/doc/results.dox

```diff
@@ -1,155 +1,14 @@
 <h1>Results</h1>
 
-The textual output the program generates is not very surprising. It just
-prints out the usual information on degrees of freedom and active cells,
-in much the same way as step-3 did:
+The created output is not very surprising. We simply see that the solution is
+periodic with respect to the left and lower boundary:
 
-@code
-Number of active cells: 1024
-Degrees of freedom: 1089
-@endcode
+<p ALIGN="center">
+@image html step-45_periodic.png
```

Comment:
Please change this to link to `http://www.dealii.org/images/steps/developer/step-45.periodic.png` (with this spelling) and the same for the other picture.